it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/spanner/SpannerResourceManager.java

```diff
@@ -310,11 +309,16 @@ public synchronized void executeDdlStatements(List<String> statements)
 
     LOG.info("Executing DDL statements '{}' on database {}.", statements, databaseId);
     try {
-      databaseAdminClient
-          .updateDatabaseDdl(instanceId, databaseId, statements, /* operationId= */ null)
-          .get();
+      // executeDdlStatments can fail for spanner staging because of failfast.
+      Failsafe.with(retryOnQuotaException())
```

Comment:
Would this need us to update the beam version/commit id in our templates?

Reply:
Resolving, as discussed, this is not part of the `apache-beam` (though the folder structure is naming it that way)